src/plugins/tables/tables.js

```diff
@@ -209,6 +209,26 @@ $document.on( "submit", ".wb-tables-filter", function( event ) {
 				$value += ( $value.length > 0 ) ? "|" : "";
 				$value += $elm.val();
 			}
+		} else if ( $elm.is( "input[type='date']" ) && $elm.val() ) {
+			if ( $elm.is( "input[name='dateFrom']" ) ) {
```

Comment:
``` input[name='dateFrom'] ``` that don't see configurable.

It will also need to be documented.

Reply:
If you have suggestion provide them.